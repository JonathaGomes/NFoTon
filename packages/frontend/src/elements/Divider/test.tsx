import { render, screen } from '../../utils/test-utils'

import Divider from '.'

describe('<Divider />', () => {
  it('should render the Divider with a light height', () => {
    const { container } = render(<Divider height="light" />)

    screen.getByRole('separator', { hidden: true })

    expect(container.firstChild).toMatchSnapshot()
  })

  it('should render the Divider with a thin height', () => {
    const { container } = render(<Divider height="thin" />)

    screen.getByRole('separator', { hidden: true })

    expect(container.firstChild).toHaveStyle({
      border: '0',
      height: '2px',
      background: '#E6E8EC'
    })
  })
})
