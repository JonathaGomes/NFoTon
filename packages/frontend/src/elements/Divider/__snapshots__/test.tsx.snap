// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> should render the Divider with a light height 1`] = `
.c0 {
  border: 0;
  height: 1px;
  background: #E6E8EC;
}

<hr
  aria-hidden="true"
  class="c0"
  height="light"
  role="separator"
/>
`;

exports[`<Section /> should render the Divider with a light height 1`] = `
.c0 {
  border: 0;
  height: 1px;
  background: #E6E8EC;
}

<hr
  aria-hidden="true"
  class="c0"
  height="light"
  role="separator"
/>
`;
